app/controllers/candidate_interface/referees_controller.rb

```diff
@@ -70,10 +70,22 @@ def update
       end
     end
 
+    def confirm_cancel
+      @application_form = current_application
+    end
+
+    def cancel
+      @referee.update!(feedback_status: 'cancelled')
+      RefereeMailer.reference_cancelled_email(current_application, @referee).deliver_later
```

Comment:
Can we send a Slack message to support? It'll be interesting to know how much this feature is used.